src/main/java/fr/esgi/al5_2/Tayarim/services/ProprietaireService.java

```diff
@@ -92,21 +99,22 @@ public ProprietaireDTO updateProprietaire(@NonNull Long id, @NonNull Proprietair
             throw new ProprietaireNotFoundException();
         }
 
-        if(proprietaireUpdateDTO.getEmail() != null && proprietaireRepository.findFirstByEmail(proprietaireUpdateDTO.getEmail()).isPresent()){
+        if((proprietaireUpdateDTO.getEmail() != null && !proprietaireUpdateDTO.getEmail().isBlank()) && proprietaireRepository.findFirstByEmail(proprietaireUpdateDTO.getEmail()).isPresent()){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '{' is not preceded with whitespace.